test/integration/v2/search.js

```diff
@@ -34,6 +34,12 @@ describe( "Search", ( ) => {
       } ).expect( "Content-Type", /json/ )
         .expect( 200, done );
     } );
+    it( "returns user by orcid", function ( done ) {
+      request( this.app ).get( "/v1/search?q=0000-0001-0002-0004" ).expect( res => {
```

Comment:
Sorry, meant to include this in the review, but this is testing the v1 implementation, not the v2 one.

Reply:
I updated this test to be v2